src/components/LoanFinding/PartnerSpotlightSection.vue

```diff
@@ -1,35 +1,41 @@
 <template>
-	<div class="tw-w-full tw-bg-secondary">
+	<div v-if="mfiRecommendationsLoans.length > 0" class="tw-w-full tw-bg-secondary">
```

Comment:
This condition was useful having this feat at the top of the lbc page; not sure if really needed here being at the bottom.

Reply:
It is needed because the loans load async, and the `LendingCategorySection` doesn't watch the `loans` prop. This was a simpler solution than adding a `watch`.